### PR TITLE
Fix typo in CODEOWNERS

### PR DESCRIPTION
**Pull request by christothes**
I was perusing the CODEOWNERS and happened to notice this.

### CODE REVIEW

**Review by pakrym — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -13,7 +13,7 @@
 /sdk/core/                                      @pakrym @KrzysztofCwalina
 
 # Extensions
-/sdk/extensions/                                @parkym
+/sdk/extensions/                                @pakrym
```
Comment (jsquire) [inline .github/CODEOWNERS:16]: We probably want to apply the `Extensions` label here as well.

**Review by jsquire — COMMENTED** [review]
LGTM

File: .github/CODEOWNERS (hunk 1 above)
Comment (christothes) [reply .github/CODEOWNERS:16]: Sorry, I was too quick to merge!

**Pull request merged**